ldpd/ldp_vty_cmds.c

```diff
@@ -0,0 +1,1134 @@
+/*
+ * Copyright (C) 2016 by Open Source Routing.
+ *
+ * This file is part of GNU Zebra.
```

Comment:
Let's fix the 'This file is part of ...'  Shouldn't it be 'This file is part of FRR'?